README.md

```diff
@@ -115,6 +115,7 @@ Please feel free to contribute after reading [contribution guidelines](CONTRIBUT
   - [AuthoritativeMovementExample](https://github.com/Relic/AuthoritativeMovementExample) - Server authoritative movement project using Forge Networking for Unity.
   - [Angry Bots Multiplayer](https://assetstore.unity.com/packages/templates/photon-angry-bots-multiplayer-showcase-1917) - Photon PUN-based multiplayer top-down shooter project.
   - [ArenaGame](https://github.com/NFMynster/ArenaGame) - Forge Networking-based multiplayer FPS game as an example project.
+  - [Entitas Sync Framework](https://github.com/RomanZhu/Entitas-Sync-Framework) - Networking framework for Entitas ECS. Targeted at slow-paced genres.
```

Comment:
I had a look at the project and it actually looks promising!
However, please don't get me wrong but I don't think it's "awesome" at this stage.
I will be keeping eye on your project for a while and hoping to see it growing.
Please feel free to open a PR again in the future when you think it's mature enough.
Maybe I could even add it in the future.

Thanks again and good luck with your project! :)